zsh-you-should-use.plugin.zsh

```diff
@@ -0,0 +1 @@
+you-should-use.plugin.zsh
```

Comment:
I was not aware this was possible. Do you have a link to documentation I could read about it?

Reply:
For sure, here’s a good entry I found https://kb.iu.edu/d/abbe

Reply:
So I understand symbolic links exist, but I didnt really think they could be committed this way into git.

I pulled your changes locally and the file does indeed get marked as a symbolic link so I guess I learned something new today